src/set.cr

```diff
@@ -203,6 +203,17 @@ struct Set(T)
     set
   end
 
+  # Addition: returns a new set containing the elements from both sets.
```

Comment:
I'd like to add something which makes it clear elements are not duplicated, and that this operation is the same as set union, *without* mentioning "alias".

Reply:
See newest commit.